app/assets/javascripts/discourse/app/components/d-editor.gjs

```diff
@@ -139,18 +139,22 @@ export default class DEditor extends Component {
       };
     });
 
-    if (this.popupMenuOptions && this.onPopupMenuAction) {
-      this.popupMenuOptions.forEach((popupButton) => {
-        if (popupButton.shortcut && popupButton.condition) {
-          const shortcut =
-            `${PLATFORM_KEY_MODIFIER}+${popupButton.shortcut}`.toLowerCase();
-          keymap[shortcut] = () => {
-            this.onPopupMenuAction(popupButton, this.newToolbarEvent());
-            return false;
-          };
-        }
-      });
-    }
+    this.popupMenuOptions?.forEach((popupButton) => {
```

Comment:
We lost the check on `this.onPopupMenuAction` here, is this ok?

Reply:
My assumption is that if we have a list of options we should have an action handler (I don't think we'd have another way of handling the action otherwise)